.eslintrc.json

```diff
@@ -66,16 +66,8 @@
     },
     "overrides": [
         {
-            // "temporarily downgraded during transition to eslint
             "files": "**",
             "rules": {
-                "@typescript-eslint/explicit-module-boundary-types": "off",
-                "@typescript-eslint/no-explicit-any": "off",
-                "@typescript-eslint/no-unsafe-assignment": "off",
-                "@typescript-eslint/no-unsafe-call": "off",
-                "@typescript-eslint/no-unsafe-member-access": "off",
-                "@typescript-eslint/no-unsafe-return": "off",
-                "@typescript-eslint/prefer-regexp-exec": "off",
                 "@typescript-eslint/restrict-template-expressions": "off"
```

Comment:
Since we're leaving this disabled, can we move it to the `rules` configuration option, rather than leaving it in an override for all files?